### DISCUSSION TITLE
jcg q20 2.4G与5G设置相同WI-FI名称之后无法打开后台

### DISCUSSION THREAD

**Discussion by uje** [immortalwrt/immortalwrt#704] [Q&A] [answered]
固件名称：immortalwrt-ramips-mt7621-jcg_q20-squashfs-factory.bin
刷完之后接了网线打开后台提示网页未找到，连接WI-FI之后获取到的IP是169.254.1.250。捅屁股reset也没用。
目前不知道该怎么办了，请问有没有办法解决这个问题？

**Comment by 1715173329** [comment]
我假设你是用的我们官方发行的固件，且版本为 snapshots 快照版。

snapshots 默认是不带 luci 界面的，需要你自行安装。
若 ip 不正确，多为你 DHCP 服务配置异常，可以尝试手动指定 IP 地址。

**Reply by uje** [reply]
已经尝试将电脑 IP 指定为 192.168.5.100，掩码指定为 255.255.255.0，也是打不开的。
后来又将电脑与路由器接到同一个路由器环境下，可以看到 jcq 20 通过 DHCP 拿到了一个 IP，使用这个 IP 打开，也是提示找不到网页。

**Reply by 1715173329** [reply]
默认网关是 192.168.1.1

**Comment by uje** [comment, marked as answer]
连接有线后，通过打开192.168.5.1，登录之后，重新设置，已经解决这个问题， 谢谢
